src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
- [ ] It would be better if we remove this line `import './App.css'` file, because  I get an error when I run the `npm run start` command. It returns "Module not found error", the application is unable to find the about file